src/components/product_detail/section1.js

```diff
@@ -0,0 +1,218 @@
+import React  from 'react'
+import styled from 'styled-components'
+import Cart from '../../images/ProductDetail/cart.png'
+import Button from '@material-ui/core/Button'
+import PlusandMinus from './PlusandMinus'
+import StarRate from './StarRate'
+import Image from './GalleryProductDetail'
+
+
+const Content = styled.div `
+    display: flex;
+    justify-content: flex-start;
+    background-color: white;
+    border-radius: 5px;
+    height: 559px;
+    width: 100%;
+    margin-bottom: 1.5rem;
+`
+
+const LeftItem = styled.div`
+    width: 50%;
+    display: flex;
+    flex-direction:column;
+    align-items: center;
+    justify-content: center;
+    padding: 50px 75px;
+    margin-right: 23px;
+`
+
+const RightItem = styled.div`
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding: 50px 0px;
+`
+
+const ProductName = styled.p`
+    font-family: Kanit;
+    align-items: center;
+    font-size: 24px;
+    color: #5b3c78;
+    margin-top: 16px;
+    margin-bottom: 10px;
+`
+
+const ContainerDetial = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+`
+
+const  ContainerStar = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    height: 40px;
+    padding: 3px 0 0 0;
+    margin-bottom: 0px;
+`
+
+
+const NameStore = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 28px;
+    margin-bottom: 15px;
+`
+
+const Subtitle  = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: 300;
+    margin-bottom: 0;
+`
+
+const IdSeller = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 25px;
+    margin-bottom: 20px;
+`
+
+const Price = styled.p`
+    font-family: Kanit;
+    font-size: 3.6rem;
+    width: 13.3rem;
+    color: #5b3c78;
+    line-height: 3.6rem;
+    margin-left: 0.8rem;
+    margin-bottom: 20px;
+`
+
+const Shipping = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 25px;
+    margin-bottom: 20px;
+`
+
+const Amount = styled.p`
+    font-family: Kanit;
+    color: #828282;
+    font-weight: normal;
+    margin-right: 45px;
+    margin-bottom: 40px;
+`
+
+const Container_PlusandMinus = styled.div`
```

Comment:
ตั้งชื่อ Component ให้เป็น Pascal Case ด้วยนะ อย่างเช่น
```suggestion
const ContainerPlusandMinus = styled.div`
```

Reply:
แก้ชื่อทุกอย่างแล้วเน้อ ขอโทดทีๆๆๆ